script/Contracts.sol

```diff
@@ -24,7 +24,7 @@ library MainnetContracts {
     address public constant PROXY_ADMIN = 0xA02A8DC24171aC161cCb74Ef02C28e3cA2204783;
 
     address public constant PROVIDER = address(123456789); // TODO: Update with deployed Provider
-    address public constant BUFFER = address(987654321); // TODO: Update with deployed buffer
+    address public constant BUFFER = address(0x45c3B59d53e2e148Aaa6a857521059676D5c0489);
```

Comment:
@danoctavian I added the BUFFER strategy address here since you said this will be the one used? Is that okay ?

Reply:
can we define a named constant separately?  with EULER_and_what_it_is

Reply:
updated in [742159e](https://github.com/yieldnest/yieldnest-vault/pull/88/commits/742159e4f34ba39e9ca27f30c84a4de4edfcf847)